packages/react/src/components/Shortcut/Shortcut.props.ts

```diff
@@ -0,0 +1,66 @@
+export interface ShortcutProps {
+  /**
+   * Specify an optional className to be added to your Shortcut
+   */
+  className?: string
+
+  /**
+   * Allowing you to change the way an HTML element is "translated" into the accessibility tree
+   */
+  ariaLabel?: string,
+
+  /**
+   * If `true`, the element will be disabled.
+   * @default false
+   */
+  disabled?: boolean
+
+  /**
+   * The click handler
+   */
+  onClick: () => void
```

Comment:
o `onClick` está indo diretamente para o `IconButtonBase`, que é baseado em um botão. Com isso, imagino que a gente precise melhorar o que esse método pode receber de parâmetro.

```suggestion
    onClick: (e?: React.MouseEvent<HTMLButtonElement>) => void
```